model/testing/pyproject.toml

```diff
@@ -0,0 +1,83 @@
+[build-system]
+build-backend = "setuptools.build_meta"
+requires = ["setuptools>=61.0", "wheel>=0.40.0"]
+
+[project]
+authors = [{email = "gridtools@cscs.ch"}, {name = "ETH Zurich"}]
+classifiers = [
+  'Development Status :: 3 - Alpha',
+  'Intended Audience :: Science/Research',
+  'License :: OSI Approved :: BSD License',
+  'Operating System :: POSIX',
+  'Programming Language :: Python',
+  'Programming Language :: Python :: 3',
+  'Programming Language :: Python :: 3 :: Only',
+  'Programming Language :: Python :: 3.10',
+  'Programming Language :: Python :: 3.11',
+  'Programming Language :: Python :: Implementation :: CPython',
+  'Topic :: Scientific/Engineering :: Atmospheric Science',
+  'Topic :: Scientific/Engineering :: Mathematics',
+  'Topic :: Scientific/Engineering :: Physics'
+]
+dependencies = [
+  # workspace members
+  'icon4py-common[netcdf]>=0.0.6',
+  # external dependencies
+  "gt4py>=1.0.4",
```

Comment:
Should we maybe install `gt4py[dace]` for testing? Please see my changes in #600.

Reply:
@egparedes I would like just to check whether you prefer to do this change as part of this PR or I should do it separately.